bazel/repositories.bzl

```diff
@@ -964,8 +965,29 @@ def _emscripten_toolchain():
             "upstream/emscripten/cache/is_vanilla.txt",
             ".emscripten_sanity",
         ]),
+        # FIXME: https://issues.redhat.com/browse/OSSM-781
         patch_cmds = [
-            "if [[ \"$(uname -m)\" == \"x86_64\" ]]; then ./emsdk install 2.0.7 && ./emsdk activate --embedded 2.0.7; fi",
+            "./emsdk install 2.0.7-upstream",
+            "./emsdk activate --embedded 2.0.7-upstream",
+            # Maistra change.
+            # Relies on the host tooling, instead of emsdk's. Requires clang with wasm32 target enabled and binaryen.
+            "rm -rf upstream/bin/*",
+            "ln -s /usr/bin/clang* upstream/bin/",
+            "ln -s /usr/bin/ll* upstream/bin/",
+            "ln -s /usr/bin/wasm* upstream/bin/",
+            # Remove unnecessary libs and files
+            "rm -rf upstream/fastcomp",
+            "rm -rf upstream/include",
+            "rm -rf upstream/lib",
+            "rm -rf upstream/libexec",
+            "rm -rf upstream/share",
+            "rm -f upstream/emscripten_config_*",
+            # Also use host's nodejs
+            "rm -rf node/12.18.1_64bit/*",
+            "mkdir node/12.18.1_64bit/bin",
+            "ln -s /usr/bin/node node/12.18.1_64bit/bin/",
+            "ln -s /usr/bin/npm node/12.18.1_64bit/bin/",
+            "ln -s /usr/bin/npx node/12.18.1_64bit/bin/",
```

Comment:
I'm afraid these hacks break `cachito`, the source management system we use for product builds